dotCMS/src/main/java/com/dotcms/graphql/datafetcher/page/PageDataFetcher.java

```diff
@@ -35,16 +38,22 @@ public Contentlet get(final DataFetchingEnvironment environment) throws Exceptio
             final HttpServletRequest request = ((DotGraphQLContext) environment.getContext())
                     .getHttpServletRequest();
 
+            final HttpServletResponse response = ((DotGraphQLContext) environment.getContext())
```

Comment:
Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6184469)